helmfile/values/gatekeeper/gatekeeper.yaml.gotmpl

```
@@ -45,6 +45,16 @@ validatingWebhookCustomRules:
       - DELETE
     resources:
       - customresourcedefinitions
+  - apiGroups:
+      - '*'
+    apiVersions:
+      - '*'
+    operations:
+      - CREATE
+      - UPDATE
+      - DELETE
+    resources:
+      - secrets
```

```suggestion
      - customresourcedefinitions
      - secrets
```
We can collapse these into one.